### ISSUE TITLE
Linux -fPIE build failure

### ISSUE THREAD

**Issue by JMS55** [LIHPC-Computational-Geometry/metis-rs#32]
I get an error on Linux when linking against some kind of .rlib from libmetris_sys gklib.o

```
,--eh-frame-hdr" "-Wl,-z,noexecstack" "-L" "/home/jms/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-o" "/home/jms/Projects/bevy/target/release/examples/meshlet-e26ebb2941c08c68" "-Wl,--gc-sections" "-pie" "-Wl,-z,relro,-z,now" "-Wl,-O1" "-Wl,--strip-debug" "-nodefaultlibs"
  = note: /usr/bin/ld: /home/jms/Projects/bevy/target/release/deps/libmetis_sys-5573fe5096f72fe6.rlib(f6cd32e89e94ec8c-gklib.o): relocation R_X86_64_32 against `.rodata' can not be used when making a PIE object; recompile with -fPIE
          /usr/bin/ld: failed to set dynamic section sizes: bad value
          collect2: error: ld returned 1 exit status
```

**Comment by JMS55** [comment]
Looks like it's the `force-optimize-vendor` feature causing the problem. Disabling it compiles fine.

Also, weirdly, leaving it on but trying to use `use-system` has the same issue. I'm not convinced that `use-system` is working.

**Comment by cedricchevalier19** [comment]
Fixed in #33.
